tests/matrix_free/matrix_vector_normal_hessians.output

```diff
@@ -0,0 +1,26 @@
+
+DEAL::test_hessians_only
+DEAL::Working with FE_Q<1>(1) and 12 dofs
+DEAL::dst FEE: 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 
+DEAL::Working with FE_Q<2>(1) and 51 dofs
+DEAL::dst FEE: 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 
+DEAL::Working with FE_Q<3>(1) and 81 dofs
+DEAL::dst FEE: 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 
+DEAL::Working with FE_DGQ<1>(1) and 22 dofs
+DEAL::dst FEE: 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 
+DEAL::Working with FE_DGQ<2>(1) and 136 dofs
+DEAL::dst FEE: 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 
+DEAL::Working with FE_DGQ<3>(1) and 232 dofs
+DEAL::dst FEE: 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 
+DEAL::Working with FE_Q<1>(2) and 23 dofs
+DEAL::dst FEE: 1465345.061 40571.23729 11202.30012 186184.9221 -92344.77470 1048204.969 49574.60308 10790.27125 666909.3801 
+DEAL::Working with FE_Q<2>(2) and 181 dofs
+DEAL::dst FEE: 0.000000000 0.000000000 -11330.88975 -5767.500760 -10780.28907 -1865.442471 0.000000000 -1626.733233 3933.131615 
+DEAL::Working with FE_Q<3>(2) and 446 dofs
+DEAL::dst FEE: 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 0.000000000 25.74078069 0.000000000 0.000000000 
+DEAL::Working with FE_DGQ<1>(2) and 33 dofs
+DEAL::dst FEE: 9211.388432 -18422.77686 9211.388432 49499.63824 -98999.27647 49499.63824 -11803.61329 23607.22657 -11803.61329 
+DEAL::Working with FE_DGQ<2>(2) and 306 dofs
```

Comment:
How did you check that these numbers are correct? Do we have a ground truth we could look at? We obviously could compare against the result by `get_hessian() / submit_hessian()` that we manually contract with the normal projector. For the `get_normal_hessian()`, we could also construct a test similar to https://github.com/dealii/dealii/blob/master/tests/matrix_free/interpolate_functions_common.h and https://github.com/dealii/dealii/blob/5eebebc98c91a77141c1aed558957f4fc2109d88/tests/matrix_free/interpolate_quadratic.cc#L59-L67 (probably by augmented what gets tested and printed there). What do you think?

Reply:
I generated the numbers with the general path and then checked the optimized case against those.

Reply:
OK, that sounds reasonable. I'm just wondering how difficult it would be to detect and find out what is right in the future. I think we don't have to do it here.

Reply:
We could print some normal Hessians after `get_normal_hessian` to detect if something was wrong with get or submit. I think it is appropriate to assume for this test that `get_hessian` and `submit_hessian` (and `normal_vector`) produce correct results as this is tested elsewhere.